tests/testfont.c

```diff
@@ -225,6 +225,10 @@ static void test_privateAddFontFile ()
 	assertEqualInt (status, Ok);
 	assert (stringsEqual (name, "Code New Roman"));
 
+	// This causes a crash in GDI+.
+#if !defined(USE_WINDOWS_GDIPLUS)
+	GdipDeleteFontFamily (families[0]);
+#endif
```

Comment:
This and the three remaining identical block below are not technically correct. `GdipGetFontCollectionFamilyList` should not return duplicates of the font family objects. However, modifying the test makes checking with ASAN/LSAN easier. If it ever gets fixed we can fix it in a way that these calls won't crash. Attempt to explicitly delete font family objects that are part of font collection can be treated as no-op.